cdk/domino_cdk/config/efs.py

```diff
@@ -28,9 +29,17 @@ class Backup:
 
     backup: Backup
     removal_policy_destroy: bool
+    enabled: bool = True
 
     @staticmethod
-    def from_0_0_0(c: dict):
+    def from_0_0_0(c: dict) -> Optional['EFS']:
+        enabled = c.get("enabled", True)
+        if "enabled" in c:
+            del c["enabled"]
```

Comment:
The pattern here (for all of the sections) should really be more like:

```
return from_loader(
  "config.thing",
  Thing(
    enable=thing.pop("enable", True),
    existing_variable=thing.pop("existing_variable"),
    [...]
)
```

Reply:
The reason I didn't go this route was to prevent having to parse a valid configuration when disabled. If you are ok with using empty sections to indicate disabled, that feels more ergonomic to me and I can make that change. thoughts?

Reply:
Yeah, let's try the empty sections. I think it will make a lot more sense and result in a cleaner (and more relevant) config for the data plane clusters.